.github/workflows/cache_server.test.yml

```diff
@@ -86,3 +86,41 @@ jobs:
           REDIS_HOST: localhost
           REDIS_PORT: ${{ job.services.redis.ports[6379] }}
           NATS_CLIENTS_URL: localhost:4222
+
+      - name: Running E2E tests
+        run: npm run test-e2e --forceExit
+        env:
+          CI: true
+          ENS_REGISTRY_ADDRESS: ${{secrets.ENS_REGISTRY_ADDRESS}}
```

Comment:
You mean it is better to get those values from `.en.ci` instead of Github secrets?

Reply:
Right. As I understand secrets are values which should not be accessible outside of CI. I don't see reason why to keep contracts private. Besides since contracts are deployed during CI we will not be able to set them ahead of tests

Reply:
@JGiter in github actions, env variable can only either be gotten from the default env variables that start with `GITHUB_` or the ones defined as `context` during execution of the workflow. since we don't want to hardcode the values during execution of the workflow, i think github secrets is right for this.